src/components/AudioPlayer/PlayerButtons/Forward/ForwardButton.container.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { playNextSong } from 'store/actions/playerActions';
+
+import ForwardButtonComponent from './ForwardButton.component';
+
+const ForwardButtonContainer = () => {
+  const songIndex = useSelector((state) => state.songsQueue.get('index'));
+  const songsQueue = useSelector((state) => state.songsQueue.get('queue'));
+  const onRepeat = useSelector((state) => state.player.get('onRepeat'));
+
+  const isDisabled = onRepeat === true ? false : songIndex + 1 >= songsQueue.size;
```

Comment:
```javascript
onRepeat === true
```

good check but can this be something else than `true|false`? if so, why? can we add a test for this somehow?

Reply:
No, it should not be other thatn true/false, i'll change that 😄 . 

About the test, sure i can try to add one.